android/src/com/eegeo/animation/ReversibleAnimatorSet.java

```diff
@@ -35,11 +35,14 @@ public float getAnimatedFraction()
 		
 		ArrayList<Animator> valueAnimators = m_animatorSet.getChildAnimations();
 		
+		float animatedFractions = 0.0f;
 		for(Animator animator : valueAnimators)
 		{
-			animatedFraction = Math.min(animatedFraction, ((ValueAnimator)animator).getAnimatedFraction());
+			animatedFractions +=((ValueAnimator)animator).getAnimatedFraction();
 		}
 		
+		animatedFraction = Math.min(animatedFraction, animatedFractions/(float)valueAnimators.size());
```

Comment:
This change makes little sense. animatedFraction is always 1.0f when Math.min is used so it is not not the minFraction and not contributing in any way unless getAnimatedFraction for a ValueAnimator ever returns more than 1.0f (which I believe is not the case).
Just using the average directly is more appropriate but beware of the potential divide by zero when there are no value animators in the set.